.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/dmap"]
 	path = lib/dmap
 	url = https://github.com/dapphub/dmap.git
+[submodule "lib/jams"]
```

Comment:
Why is this a submodule when jams.js is a dependency?

Reply:
Originally I had added it as a dependency but noticed two issues: 1) the bug you mentioned and 2) the version was very outdated. Thought it might be easier to work as a sub module since jams will likely keep getting feature adds. 

I forgot to uninstall the npm version. I will revert all the import changes if the npm published version is working.

Reply:
addressed